Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>.gradle\n.idea\nlocal.*\nbuild.gradle\ngradle/wrapper/gradle-wrapper.properties\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 144fe4d29d92e91c87546d25734e03285eff9333)
+++ b/.gitignore	(date 1621690175964)
@@ -1,4 +1,4 @@
-.gradle
+#.gradle
 .idea
 local.*
 build.gradle
